aucoalesce/normalizations.yaml

```diff
@@ -1001,7 +1072,9 @@ normalizations:
   - <<: *macro-user-session
     record_types: USER_END
     action: ended-session
-    ecs: *ecs-auth
```

Comment:
This changes USER_START/USER_END messages from _event.category:_ `authentication` to `session`.

I wonder if I should keep the former, `event.category: [authentication, session]`

WDYT @andrewstucki ?

Reply:
I'm leaning toward only session since there are separate events for authentication. If we had session earlier I assume that's the way we would have done it. It's a minor breaking change though.

Reply:
I'm fine with that change. Also, just in the interest of spreading the knowledge around, main place I could find a reference to `AUDIT_USER_LOGIN` is when `login` from shadow-utils returns successfully.

https://github.com/shadow-maint/shadow/blob/ae169c40468bc9d016016880613c01aca6050b60/src/login.c#L837

`AUDIT_USER_START` is predominantly an artifact of PAM and is called in `pam_open_session`:

https://github.com/linux-pam/linux-pam/blob/491e5500b6b3913f531574208274358a2df88659/libpam/pam_audit.c#L148-L151

Both work hand-in-hand, as you can see the invocation of `pam_open_session` here:

https://github.com/shadow-maint/shadow/blob/ae169c40468bc9d016016880613c01aca6050b60/src/login.c#L885

Seems to mirror exactly what's described in the [audit framework docs](https://github.com/linux-audit/audit-documentation/wiki/SPEC-User-Login-Lifecycle-Events)

> The next event can be AUDIT_USER_ROLE_CHANGE depending on whether a MAC framework that has the notion of roles is in use. It records what role has been assigned to the login session. This event is followed by AUDIT_USER_LOGIN to signify that the session that is being created is interactive. This is followed by AUDIT_USER_START to denote that the session has started.